internal/repository/user.go

```diff
@@ -0,0 +1,122 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/dilyara4949/flight-booking-api/internal/domain"
+	"github.com/google/uuid"
+)
+
+type userRepository struct {
+	db             *sql.DB
+	contextTimeout time.Duration
+}
+
+func NewUserRepository(db *sql.DB, timeout time.Duration) domain.UserRepository {
```

Comment:
there is no sense to return an interface

Reply:
in that case, interface do not needed at all, isn't it?

Reply:
the struct methods and interface methods are the same, does it mean that in such cases i dont need interface?